Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Call.hs

```diff
@@ -189,11 +191,12 @@ directCallStatusCallback callSid dialCallStatus recordingUrl_ callDuratioExotel
           _ <- updateCallStatus callStatus.id newCallStatus Nothing callDuration
           throwError CallStatusDoesNotExist
         else updateCallStatus callStatus.id newCallStatus Nothing callDuration
+  DOC.sendCallDataToKafka (Just "EXOTEL") (Id callStatus.entityId) "EXOTEL" (Just callSid) (Just (show dialCallStatus)) System Nothing
```

Comment:
Is entityId in this corresponds to rideId ?

Reply:
Why EXOTEL as callType ?

Reply:
What is personId and why Nothing?

Reply:
1. Yes
2. Since log is from exotel api
3. For exotel api calls we don't have perosnId